packages/web-components/src/components/cbp-panel/cbp-panel.stories.tsx

```diff
@@ -62,7 +62,7 @@ const PanelTemplate = ({ role, headingLevel, header, headerId, content, ariaLabe
         variant="heading-lg"
         ${headerId ? `id="${headerId}"` : ''}
       >
-        ${showIcon ? '<cbp-icon name="star" sx=\'{"margin-right":"var(--cbp-space-4x)"}\'></cbp-icon>' : ''}${header}
+        ${showIcon ? '<cbp-icon name="star" size="1.5rem" sx=\'{"margin-right":"var(--cbp-space-4x)", "vertical-align":"text-top"}\'></cbp-icon>' : ''}${header}
```

Comment:
use a token for size. And can you use backticks here so you don't have to escape the single quotes? (or it won't let you since you're already in a template literal?)